toolkit/tools/imagegen/configuration/packageRepo.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
We did the various config.go files as all lower case (not sure WHY, but we did). Should be `packagerepo.go` to conform.

Reply:
Fixed